cfgov/v1/models/home_page.py

```diff
@@ -18,16 +16,10 @@ class HomePage(CFGOVPage):
 
     def get_template(self, request, *args, **kwargs):
         try:
-            if flag_enabled("SPANISH_HOMEPAGE"):
-                return {
-                    "en": "v1/home_page/home_page.html",
-                    "es": "v1/home_page/home_page.html",
-                }[self.language]
-            else:
-                return {
-                    "en": "v1/home_page/home_page.html",
-                    "es": "v1/home_page/home_page_legacy.html",
-                }[self.language]
+            return {
+                "en": "v1/home_page/home_page.html",
+                "es": "v1/home_page/home_page.html",
+            }[self.language]
```

Comment:
For simplicity, we could remove this entire method in favor of a

```py
template = "v1/home-page/home_page.html"
```

If someone were to create a HomePage with another language, they'd still get a (slightly less helpful) error message when [this import statement](https://github.com/cfpb/consumerfinance.gov/blob/e2f151a3c1cfdb7435fee5b6a6bbd018886c62e0/cfgov/v1/jinja2/v1/home_page/home_page.html#L19) failed.

If we did that we could remove [these unit tests](https://github.com/cfpb/consumerfinance.gov/blob/e2f151a3c1cfdb7435fee5b6a6bbd018886c62e0/cfgov/v1/tests/models/test_home_page.py) entirely as well.